databaseconnect.py

```diff
@@ -100,7 +97,7 @@ def get_chat_response():
     total_chat_records = res[0]
     import random
     chat_id = random.randint(1, total_chat_records+1)
-    cur.execute(f"SELECT sentence FROM chat_table WHERE id = {chat_id}")
+    cur.execute("SELECT sentence FROM chat_table WHERE id = '{}'".format(chat_id))
```

Comment:
Align all changes to follow the same pattern, I see that this uses format option, while the others do not.

Reply:
Done 👍

Reply:
Kindly take a look at [this](https://realpython.com/prevent-python-sql-injection/).
Using _either of_ `f-strings`, `.format()` or `%` **WILL** open the code up to SQL injections.

Take this example and see if it makes sense:
```python
# Never do this -- insecure!
symbol = 'RHAT'
c.execute("SELECT * FROM stocks WHERE symbol = '%s'" % symbol)

# Do this instead
t = ('RHAT',)
c.execute('SELECT * FROM stocks WHERE symbol=?', t)
print(c.fetchone())

# Larger example that inserts many records at a time
purchases = [('2006-03-28', 'BUY', 'IBM', 1000, 45.00),
             ('2006-04-05', 'BUY', 'MSFT', 1000, 72.00),
             ('2006-04-06', 'SELL', 'IBM', 500, 53.00),
            ]
c.executemany('INSERT INTO stocks VALUES (?,?,?,?,?)', purchases)
```
Source: [Python Docs `sqlite3`](https://docs.python.org/3/library/sqlite3.html)
@Pihu1998